src/components/MyCalculator.js

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react';
+
+function Calculator() {
+  const [displayValue, setDisplayValue] = useState('0');
+
+  const handleNumberClick = (value) => {
+    setDisplayValue((prevValue) => (prevValue === '0' ? String(value) : prevValue + String(value)));
+  };
+
+  const handleClearClick = () => {
+    setDisplayValue('0');
+  };
+
+  const handleEqualsClick = () => {
+    // to be implemented in the next project
+  };
+
+  return (
+    <div className="calculator">
+      <div className="calculator-display">{displayValue}</div>
+      <div className="calculator-keypad">
+        <div className="calculator-keypad-row">
+          <button type="button" onClick={() => handleNumberClick('AC')}>
+            AC
+          </button>
+          <button type="button" onClick={() => handleNumberClick('+/-')}>
+            +/-
+          </button>
+          <button type="button" onClick={() => handleNumberClick('%')}>
+            %
+          </button>
+          <button
+            className="btn-color"
+            type="button"
+            onClick={() => handleNumberClick('÷')}
```

Comment:
- [ ] Please make sure to create at least one child component for the Calculator component as requested in the project requirements. (I would suggest separating the buttons code in a different component and importing it here in the Calculator component).

![image](https://user-images.githubusercontent.com/83803414/234105771-d4c9e88d-2e24-4490-9bae-563b9fbdc2e0.png)